src/views/Home/index.jsx

```diff
@@ -0,0 +1,125 @@
+  },
+});
+
+class App extends Component {
+  render() {
+    const { classes } = this.props;
+
+    return (
+      <div className={classes.root}>
+        <Grid
+          container
+          spacing={16}
+          direction="row"
+          justify="center"
+          alignItems="center">
+          <Grid item xs={12}>
+            <div style={{ display: 'flex', flexDirection: 'row' }}>
+              <div style={{ flex: 1 }} />
+              <Card className={classes.mainCard} raised>
+                <Typography
+                  variant="display4"
+                  className={classes.whiteText}
+                  align="center">
+                  BugsAhoy
+                </Typography>
+                <Typography
+                  variant="display1"
+                  align="center"
+                  className={classes.whiteText}>
+                  A good first issue finder
+                </Typography>
+              </Card>
+              <div style={{ flex: 1 }} />
+            </div>
+          </Grid>
+          <Grid item xs={12}>
+            <Grid
+              container
+              spacing={16}
+              direction="row"
+              justify="center"
+              alignItems="center">
+              <Grid item xs={3}>
+                <Link to="/projects" style={{ textDecoration: 'none' }}>
+                  <Card className={classes.buttonCard}>
+                    <Typography
+                      variant="headline"
+                      align="center"
+                      className={classes.text}>
+                      Projects Information
+                    </Typography>
+                    <Divider />
+                    <Typography
+                      variant="caption"
+                      align="center"
+                      className={classes.text}>
+                      The description, website, as well as communication
+                      channels used by particular projects.
+                    </Typography>
+                  </Card>
+                </Link>
+              </Grid>
+              <Grid item xs={3}>
+                <Link to="/bugs" style={{ textDecoration: 'none' }}>
+                  <Card className={classes.buttonCard}>
+                    <Typography
+                      variant="headline"
+                      align="center"
+                      className={classes.text}>
+                      Bugs{' '}
+                    </Typography>
+                    <Divider />
+                    <Typography
+                      variant="caption"
+                      align="center"
+                      className={classes.text}>
+                      Bugs for you to solve! Filterable and customizable list
+                      based on your skills and interests.
+                    </Typography>
+                  </Card>
+                </Link>
+              </Grid>
+            </Grid>
+          </Grid>
+        </Grid>
+      </div>
```

Comment:
I don't think we should worry about adding theme colors for now. We should focus on the layout first. Also, we should use `className` instead of `style` whenever possible to separate styling from jsx.

I think the styling on the class name `root` as well as `<div style={{ flex: 1 }} />` is messing up with the layout. When I resize the screen I get:

<img width="432" alt="screen shot 2018-05-21 at 9 55 04 am" src="https://user-images.githubusercontent.com/3766511/40311747-b62c92ec-5cde-11e8-9f72-2f9d154ca677.png">

What do you think if we change the jsx to be something like this:

```jsx
<Fragment>
  <Typography
    variant="display3"
    align="center">
    BugsAhoy
  </Typography>
  <Typography
    variant="display1"
    align="center">
    A good first issue finder
  </Typography>
  <Grid container>
    <Grid item xs={12} md={6}>
      <Card className={classes.buttonCard}>
        <CardContent>
          <Link to="/projects" className={classes.link}>
            <Typography variant="headline">
              Projects
            </Typography>
            <Typography component="p">
              The description, website, as well as communication
              channels used by particular projects.
            </Typography>
          </Link>
        </CardContent>
      </Card>
    </Grid>
    <Grid item xs={12} md={6}>
      <Card className={classes.buttonCard}>
        <CardContent>
          <Link to="/projects" className={classes.link}>
            <Typography variant="headline">
              Bugs
            </Typography>
            <Typography component="p">
              Bugs for you to solve! Filterable and customizable list
              based on your skills and interests.
            </Typography>
          </Link>
        </CardContent>
      </Card>
    </Grid>
  </Grid>
</Fragment>
```

And have the styles be:

```js
buttonCard: {
  margin: theme.spacing.unit * 3,
  minHeight: 300,
},
link: {
  textDecoration: 'none',
},
```
<img width="517" alt="screen shot 2018-05-21 at 10 15 20 am" src="https://user-images.githubusercontent.com/3766511/40312129-f289da78-5cdf-11e8-9680-d55103652bcd.png">